package.json

```diff
@@ -74,8 +74,8 @@
     "lint:scss": "stylelint 'packages/**/src/**/*.scss' --syntax scss",
     "pretest": "npm run lint",
     "postinstall": "link-parent-bin && npm run compile && npm run bootstrap:hoist",
-    "publish": "npm whoami && check-installed-dependencies && npm run compile && npm test && lerna publish",
-    "postpublish": "npm run docs && git add --all && git commit -m 'Updated README.md'",
+    "release": "npm whoami && check-installed-dependencies && npm run compile && npm test && lerna publish",
```

Comment:
If you have an npm script named `publish`, lerna will try to run it during `lerna publish` lifecycle. Renaming script to `release` to avoid this.